### PR TITLE
cn trans

### PR DESCRIPTION
**Pull request by LuminescenceRecordings**
Discord: LRecords#3442

### CODE REVIEW

**Review by randomwalk266 — CHANGES_REQUESTED** [review]
thanks for your contribution. i have translated some article,  such as faqs and how-do-i-get-a-wallet. in order to avoid repetitive work, you can translate other article first.

**Comment by LuminescenceRecordings** [comment]
> thanks for your contribution. i have translated some article, such as faqs and how-do-i-get-a-wallet. in order to avoid repetitive work, you can translate other article first.

Got it, I do the rest of articles/

**Review by randomwalk266 — APPROVED** [review]

**Comment by LuminescenceRecordings** [comment]
2 titles of up level mistake,
'使用狗狗币时候我的隐私有保护吗？'   remove   '侯'
'那些慈善机构接受狗狗币？'   change '那' to '哪'
截屏2022-08-22 04 59 40

File: content/zh-cn/dogepedia/articles/dogecoin-developers.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+2014年，Billy和Jackson离开了这个项目。另一组开发人员取代了他们，成为该项目的维护者。这个团队多年来一直在发生变化，并协调了40多个贡献者的成果。新的贡献者可以在Github上公开的项目中加入Dogecoin Core的开发，[此处](https://github.com/dogecoin/dogecoin)。
 
-Development of the overall ecosystem is also crucial - as if the aim of the reference implementation is to build a solid and stable foundation, innovation can happen through services built on top of it (and at times, in synergy with it). There are many solo developers, organizations, and companies building services on top of the existing Dogecoin Core codebase, including payment channel functionality, experimental support for NFTs, and tipping services. The Dogecoin Foundation repository can be found on Github [here](https://github.com/dogecoinfoundation).
+整个生态系统的发展也至关重要——就好像参考实现的目的是为了建立一个坚实而稳定的基础，创新可以通过建立在其上的服务产生(有时，与之协同)。有许多独立开发者、组织和公司在现有Dogecoin Core的代码基础上构建服务，包括支付通道功能、对NFT的实验性支持和打赏服务。狗狗币基金会的项目可以在Github上找到，[这里](https://github.com/dogecoinfoundation)。
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/dogecoin-developers.md:16]: 1. Dogecoin Core -> 狗狗币核心
2. [这里] -> [请查看这里]

File: content/zh-cn/dogepedia/articles/dogecoin-developers.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-You do not need any authorization or permission to start contributing to Dogecoin Core or to build new services on top of it. If you are interested in becoming a Dogecoin developer, learn more [here](/dogepedia/articles/becoming-a-dogecoin-developer).
\ No newline at end of file
+你不需要任何授权或许可就可以开始为Dogecoin Core做出贡献或在其上构建新的服务。如果你有兴趣成为一名狗狗币开发者，[在这里](/dogepedia/articles/becoming-a-dogecoin-developer)了解更多。
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/dogecoin-developers.md:18]: 1. Dogecoin Core -> 狗狗币核心
2. /dogepedia/articles/becoming-a-dogecoin-developer ->/zh-cn/dogepedia/articles/becoming-a-dogecoin-developer/
3. 了解更多 -> 可以了解更多

**Review by bobdos — CHANGES_REQUESTED** [review]

**Comment by LuminescenceRecordings** [comment]
@bobdos Nice check, well done

**Review by bobdos — APPROVED** [review]
all good and it's ready to go.

many thanks for your contribution.

this is the way.

cheers.

**Pull request merged**